routes/dist_tags.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+const co = require('co')
+const r = require('koa-router')()
+const npm = require('../lib/npm')
+const config = require('../config')
+const job = require('../lib/job')
+const _ = require('lodash')
+const parse = require('co-body')
+const middleware = require('../middleware')
+
+let updateDistTags = job(co.wrap(function * (name, tags) {
+  let key = `dist-tags/${name}`
+  let current = yield config.storage.getJSON(key)
```

Comment:
And this is called every time the CLI updates, so it's good to be mindful of it.